server/webhook.go

```diff
@@ -267,6 +268,63 @@ func (p *Plugin) postPullRequestEvent(event *github.PullRequestEvent) {
 	}
 }
 
+func (p *Plugin) handlePRDescriptionMentionNotification(event *github.PullRequestEvent) {
+	action := event.GetAction()
+	if action != "opened" {
+		return
+	}
+
+	body := event.GetPullRequest().GetBody()
+
+	mentionedUsernames := parseGitHubUsernamesFromText(body)
+
+	message, err := renderTemplate("pullRequestMentionNotification", event)
+	if err != nil {
+		mlog.Error("failed to render template", mlog.Err(err))
+		return
+	}
+
+	post := &model.Post{
+		UserId:  p.BotUserID,
+		Message: message,
+		Type:    "custom_git_mention",
+	}
+
+	for _, username := range mentionedUsernames {
+		// Don't notify user of their own comment
+		if username == event.GetSender().GetLogin() {
+			continue
+		}
+
+		// Notifications for pull request authors are handled separately
+		if username == event.GetPullRequest().GetUser().GetLogin() {
+			continue
+		}
+
+		userId := p.getGitHubToUserIDMapping(username)
+		if userId == "" {
+			continue
+		}
+
+		if event.GetRepo().GetPrivate() && !p.permissionToRepo(userId, event.GetRepo().GetFullName()) {
+			continue
+		}
+
+		channel, err := p.API.GetDirectChannel(userId, p.BotUserID)
+		if err != nil {
+			continue
+		}
+
+		post.ChannelId = channel.Id
+		_, err = p.API.CreatePost(post)
+		if err != nil {
+			mlog.Error("Error creating mention post: " + err.Error())
+		}
+		p.sendRefreshEvent(p.BotUserID)
```

Comment:
Maybe I am understanding something wrong, but shouldn't the refresh event be sent to the userID, instead of to the bot?

Reply:
Good catch @larkox. Yes that should be
```suggestion
		p.sendRefreshEvent(userID)
```
instead.

@nickmisasi13 Would you also please fix that issue at https://github.com/mattermost/mattermost-plugin-github/blob/f3ed30562256da481badc929f6fbda50172c49fb/server/webhook.go#L728